src/main/resources/avro/referencemethods.avdl

```diff
@@ -166,17 +172,17 @@ Reference getReference(
     */
     string id) throws GAException;
 
-/****************  /references/{id}/bases  *******************/
```

Comment:
That call is still there, sort of: it's been re-named to the `/sequence/{id}` endpoint and the `getSequence()` method. I could have a copy under the old name though. Would that make sense?